src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,33 @@
-export const TodoInfo = () => {};
+import cn from 'classnames';
+import { UserInfo } from '../UserInfo';
+
+type User = {
+  id: number;
+  name: string;
+  username: string;
+  email: string;
+};
+
+type Todo = {
+  id: number;
+  title: string;
+  completed: boolean;
+  userId: number;
+  user: User | null;
+};
+
+type Props = {
+  todo: Todo;
+};
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => (
+  <article
+    className={cn('content', 'box', {
+      'is-primary': todo.completed,
+    })}
+    data-id={todo.id}
+  >
+    <h3 className="TodoInfo__title">{todo.title}</h3>
+    <UserInfo user={todo.user} />
```

Comment:
Ensure that the `UserInfo` component can handle a `null` user gracefully, as the `user` property in the `Todo` type can be `null`. This might involve checking for `null` before rendering or handling it within the `UserInfo` component.